File: TelerikUIExamples/TelerikUIExamples/OptionSection.m
//
//  OptionSection.m
//  TelerikUIExamples
//
//  Copyright (c) 2015 Telerik. All rights reserved.
//

#import "OptionSection.h"

@implementation OptionSection

- (instancetype)initWithTitle:(NSString*)title
{
    self = [self init];
    if (self) {
        self.title = title;
        self.items = [NSMutableArray new];
    }
    return self;
}

@end

File: TelerikUIExamplesInSwift/TelerikUIExamplesInSwift/OptionSection.swift
//
//  OptionSection.swift
//  TelerikUIExamplesInSwift
//
//  Copyright (c) 2015 Telerik. All rights reserved.
//

import UIKit

class OptionSection: NSObject {
   
    var title = ""
    
    var items = NSMutableArray()
    
    var selectedOption: Int = 0

    init(Text:NSString) {
        self.title = Text
    }
}
